Update no_sep selection logic in lab and radiology models

presidenwashil opened the pull request:
Replaced direct bridging_sep.no_sep references with a COALESCE SQL expression that selects no_sep based on patient status and jnspelayanan, improving accuracy for both PeriksaLab and PeriksaRadiologi models. Also updated search conditions and raw columns to use the new logic.

presidenwashil committed:
Update no_sep selection logic in lab and radiology models

Replaced direct bridging_sep.no_sep references with a COALESCE SQL expression that selects no_sep based on patient status and jnspelayanan, improving accuracy for both PeriksaLab and PeriksaRadiologi models. Also updated search conditions and raw columns to use the new logic.

Rizky92 commented:
```php
->leftJoin('bridging_sep', fn (JoinClause $join) => $join
    ->on('reg_periksa.no_rawat', '=', 'bridging_sep.no_rawat')
    ->on('reg_periksa.status_lanjut', '=', DB::raw('(if(bridging_sep.jnspelayanan = "1", "Ranap", "Ralan"))'))
)

presidenwashil committed:
Refactor no_sep join logic in lab and radiology models

Replaces complex subqueries for 'no_sep' with a direct join to bridging_sep using a conditional join on jnspelayanan and status_lanjut. This simplifies the query and improves maintainability in both PeriksaLab and PeriksaRadiologi models.
